notes/eligibility-summary.vbs

```diff
@@ -15815,120 +15815,15 @@ class emer_eligibility_detail
-                            If manual_hh_count > 10 Then emer_inelig_fpg_limit = 9345 + ((manual_hh_count-10) * 787)
-					    End if
-                    Else
-                        If manual_hh_count = 1 Then emer_inelig_fpg_limit = 2265
-                        If manual_hh_count = 2 Then emer_inelig_fpg_limit = 3052
-                        If manual_hh_count = 3 Then emer_inelig_fpg_limit = 3838
-                        If manual_hh_count = 4 Then emer_inelig_fpg_limit = 4625
-                        If manual_hh_count = 5 Then emer_inelig_fpg_limit = 5412
-                        If manual_hh_count = 6 Then emer_inelig_fpg_limit = 6198
-                        If manual_hh_count = 7 Then emer_inelig_fpg_limit = 6985
-                        If manual_hh_count = 8 Then emer_inelig_fpg_limit = 7772
-                        If manual_hh_count = 9 Then emer_inelig_fpg_limit = 8558
-                        If manual_hh_count = 10 Then emer_inelig_fpg_limit = 9345
-                        If manual_hh_count > 10 Then emer_inelig_fpg_limit = 9345 + ((manual_hh_count-10) * 787)
-                    End if
-
-                    emer_inelig_fpg_limit = FormatNumber(emer_inelig_fpg_limit, 2, -1, 0, -1)
-				End If
-
-				If emer_program = "EA" Then
-					If household_size = 1 Then emer_fpg_limit = 2430
-					If household_size = 2 Then emer_fpg_limit = 3287
-					If household_size = 3 Then emer_fpg_limit = 4143
-					If household_size = 4 Then emer_fpg_limit = 5000
-					If household_size = 5 Then emer_fpg_limit = 5857
-					If household_size = 6 Then emer_fpg_limit = 6713
-					If household_size = 7 Then emer_fpg_limit = 7570
-					If household_size = 8 Then emer_fpg_limit = 8427
-					If household_size = 9 Then emer_fpg_limit = 9283
-					If household_size = 10 Then emer_fpg_limit = 10140
-					If household_size > 10 Then emer_fpg_limit = 10140 + ((household_size - 10) * 857)
-					emer_fpg_limit = FormatNumber(emer_fpg_limit, 2, -1, 0, -1)
-
-					If manual_hh_count = 1 Then emer_inelig_fpg_limit = 2430
-					If manual_hh_count = 2 Then emer_inelig_fpg_limit = 3287
-					If manual_hh_count = 3 Then emer_inelig_fpg_limit = 4143
-					If manual_hh_count = 4 Then emer_inelig_fpg_limit = 5000
-					If manual_hh_count = 5 Then emer_inelig_fpg_limit = 5857
-					If manual_hh_count = 6 Then emer_inelig_fpg_limit = 6713
-					If manual_hh_count = 7 Then emer_inelig_fpg_limit = 7570
-					If manual_hh_count = 8 Then emer_inelig_fpg_limit = 8427
-					If manual_hh_count = 9 Then emer_inelig_fpg_limit = 9283
-					If manual_hh_count = 10 Then emer_inelig_fpg_limit = 10140
-					If manual_hh_count > 10 Then emer_inelig_fpg_limit = 10140 + ((manual_hh_count - 10) * 857)
-					emer_inelig_fpg_limit = FormatNumber(emer_inelig_fpg_limit, 2, -1, 0, -1)
-				End If
+				'Determine the 200% FPG for household_size
+				Call determine_200_percent_of_FPG(emer_program, emer_appl_date, household_size, emer_fpg_limit)
```

Comment:
Lines 15819 and 15824, the emer_program in the function parameters need to be changed to "EGA" and "EA", right?

Reply:
Discussed at P&I, function will work since emer_program variable in script would be set to equal "EA" or "EGA" when the function is called.